### PR TITLE
adding report category & panel date type TM-3929

### PR DESCRIPTION
**Pull request by SidelineCory24**
Dual Merge: 
- [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2439)
- [Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/335)

To-do:
- [x]  return panel date type
- [x] return panel meeting seqnum
- [x] return report category code
- [x] return report category text

### CODE REVIEW

File: talentmap_api/fsbid/services/agenda.py [hunk 1]
```diff
@@ -247,8 +251,11 @@ def fsbid_single_agenda_item_to_talentmap_single_agenda_item(data, remarks={}):
         "id": data.get("aiseqnum", None),
         "remarks": services.parse_agenda_remarks(data.get("aicombinedremarktext") or "", remarks),
         "panel_date": ensure_date(pydash.get(data, "Panel[0].pmddttm", None), utc_offset=-5),
+        "panel_date_type": pydash.get(data, "Panel[0].pmtcode") or None,
+        "panel_meeting_seq_num": str(int(pydash.get(data, "Panel[0].pmseqnum"))) or None,
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/agenda.py:255]: if `pydash.get(data, "Panel[0].pmseqnum")` returns a `None` you'll get an error

Comment (SidelineCory24) [reply talentmap_api/fsbid/services/agenda.py:255]: updated to handle if `None` is returned

**Review by elizabeth-jimenez — APPROVED** [review]

**Comment by elizabeth-jimenez** [comment]
@SidelineCory24 you can merge this pr and start a new one for the debugging of ingesting AI at AIM, up to u!

**Comment by SidelineCory24** [comment]
> @SidelineCory24 you can merge this pr and start a new one for the debugging of ingesting AI at AIM, up to u!

a fix was not needed on the BE, turned out to be a mock issue that should not arise in dev1

**Comment by scott062** [comment]
Are we ready to merge this?

**Pull request merged**